metrics/cli/docs/features/diagrams/update_feature_integration_component_diagram.puml

```diff
@@ -0,0 +1,24 @@
+@startuml update_feature_integration_component_diagram
```

Comment:
🔨 I believe it would be great to show how and in which place we are going to integrate this feature. It means that it would be great to show any key components of this feature (like Metrics CLI, Flutter Service/CLI, etc.) and show the existing relationships and a new one. Overall, it would be great to add a bit more context to this diagram to make it display the place we are going to integrate this feature and the components of the application we are going to add/modify.